terraform/file_scanning.tf

```
@@ -1,16 +1,14 @@
 module "s3_scan_objects" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.9//S3_scan_object"
+  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.1"
 
-  product_name          = "share-files-securely"
   s3_upload_bucket_name = module.share_files_securely_bucket.s3_bucket_id
 
   billing_tag_value = var.billing_code
 }
 
 module "s3_scan_objects_dev" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.9//S3_scan_object"
+  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.1"
```

Could you bump this to the latest and use the new input I just added.  This PR made me realize the module had a bug that would only let it be used once per account 😅 
```suggestion
  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.5"
```